includes/Modules/Site_Verification.php

```diff
@@ -37,13 +37,62 @@
 final class Site_Verification extends Module implements Module_With_Scopes {
 	use Module_With_Scopes_Trait;
 
+	/**
+	 * Meta site verification type.
+	 */
+	const VERIFICATION_TYPE_META = 'META';
+
+	/**
+	 * File site verification type.
+	 */
+	const VERIFICATION_TYPE_FILE = 'FILE';
+
+	/**
+	 * Callback invoked after serving verification file.
+	 *
+	 * @var \Closure
```

Comment:
This should also have a `@since` annotation (all classes, properties, methods, almost everything actually).

Reply:
Done!